### PR TITLE
Update README.dev.rst

### PR DESCRIPTION
**Pull request by jspaaks**
**List of related issues or pull requests**

Refs: #261

**Describe the changes made in this pull request**

Updated docker build instructions

**Instructions to review the pull request**

[README.dev.rst](https://github.com/fair-software/howfairis/blob/261-docker-images-instructions/README.dev.rst)

### CODE REVIEW

File: README.dev.rst [hunk 1]
```diff
@@ -160,10 +160,26 @@ To build the image, run:
 .. code:: shell
 
     docker build -t fairsoftware/howfairis .
```
Comment (fdiblen) [inline README.dev.rst:162]:
```suggestion
    docker build -t fairsoftware/howfairis:latest .
```

File: README.dev.rst [hunk 2]
```diff
@@ -160,10 +160,26 @@ To build the image, run:
...
 
 To push the image to DockerHub, run:
 
 .. code:: shell
 
     # (requires credentials)  
-    docker push fairsoftware/howfairis
+    docker login
+    docker push fairsoftware/howfairis:latest
+    docker push fairsoftware/howfairis:${VERSION}
```
Comment (fdiblen) [inline README.dev.rst:182]:
```suggestion
    docker push fairsoftware/howfairis:${VERSION}
    docker push fairsoftware/howfairis:latest    
```

**Review by fdiblen — CHANGES_REQUESTED** [review]
I proposed very minor changes. Thanks @jspaaks :fish:

**Review by fdiblen — APPROVED** [review]
Good to go! :1234:

**Comment by jspaaks** [comment]
Thanks for the suggestions

**Pull request merged**